tiledb/cloud/notebook.py

```diff
@@ -337,7 +339,9 @@ def _write_notebook_to_array(
 
     contents_as_array = numpy.array(bytearray(ipynb_file_contents, CHARACTER_ENCODING))
 
-    with tiledb.open(tiledb_uri, mode="w", ctx=ctx) as arr:
+    vfs = tiledb.VFS(tiledb.cloud.Ctx().config())
+
+    with vfs.open(tiledb_uri, mode="w", ctx=ctx) as arr:
```

Comment:
`vfs.open` returns a FileIO object, not an array, so the assignments below will not work -- this needs to be `tiledb.open` to return an array object.

Reply:
sorry @ihnorton it looks like i was rushing through this a bit :(